scripts/build

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-cd $(dirname $0)
-cd ..
-docker build -f ./package/Dockerfile -t YOUR_DOCKERHUB_REPO_HERE/plugin-server:v0 --push .
```

Comment:
removed in favor of calling this `scripts/package`, since `scripts/build` generally corresponds to `go build` in Rancher repos